Changelog.md

```diff
@@ -7,6 +7,7 @@
 - Bug Fix: Prevent assigning inactive graders to a group (#6801)
 - Added new API routes for the `create` and `destroy` actions of SectionsController (#6873)
 - Display error message in real-time when running automated tests, rather than waiting for page refresh (#6878)
+- - Added `destroy` action to `controllers/api/assignments_controller` (#6880)
```

Comment:
There's an extra `-` here. Also, please do a pull from master; we've made a new minor release, so you'll need to add this to the new `[unreleased]` section